INFO 2023-10-01 20:10:12,050 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5140 trigger_reload
INFO 2023-10-01 20:10:12,478 Watching for file changes with StatReloader autoreload 5189 run_with_reloader
INFO 2023-10-01 20:10:15,478 "GET /blog/authors/add_new_author/ HTTP/1.1" 200 1859 basehttp 5189 log_message
WARNING 2023-10-01 20:10:15,557 "GET /static/hwapp1/css/style.css HTTP/1.1" 404 1819 basehttp 5189 log_message
INFO 2023-10-01 20:10:28,624 Add new author name='asdsgsdgs', surname='a' views 5189 add_author_form
INFO 2023-10-01 20:10:28,631 "POST /blog/authors/add_new_author/ HTTP/1.1" 200 21 basehttp 5189 log_message
INFO 2023-10-01 20:10:42,144 "GET /blog/authors/add_new_author/ HTTP/1.1" 200 1859 basehttp 5189 log_message
INFO 2023-10-01 20:10:44,561 "GET /blog/ HTTP/1.1" 200 1170 basehttp 5189 log_message
WARNING 2023-10-01 20:10:44,625 "GET /static/hwapp1/css/style.css HTTP/1.1" 404 1819 basehttp 5189 log_message
INFO 2023-10-01 20:10:46,491 "GET /blog/authors/ HTTP/1.1" 200 223 basehttp 5189 log_message
INFO 2023-10-01 20:11:33,446 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5189 trigger_reload
INFO 2023-10-01 20:11:33,856 Watching for file changes with StatReloader autoreload 5220 run_with_reloader
INFO 2023-10-01 20:14:55,946 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5220 trigger_reload
INFO 2023-10-01 20:14:56,378 Watching for file changes with StatReloader autoreload 5286 run_with_reloader
INFO 2023-10-01 20:14:56,645 "GET /blog/authors/ HTTP/1.1" 200 1495 basehttp 5286 log_message
WARNING 2023-10-01 20:14:56,696 "GET /static/hwapp1/css/style.css HTTP/1.1" 404 1819 basehttp 5286 log_message
INFO 2023-10-01 20:16:56,669 /home/olgatorres/PycharmProjects/Django/testproject/blog/urls.py changed, reloading. autoreload 5286 trigger_reload
INFO 2023-10-01 20:16:57,367 Watching for file changes with StatReloader autoreload 5499 run_with_reloader
INFO 2023-10-01 20:18:23,461 "GET /random/ HTTP/1.1" 200 1503 basehttp 5499 log_message
WARNING 2023-10-01 20:18:23,533 "GET /static/hwapp1/css/style.css HTTP/1.1" 404 1819 basehttp 5499 log_message
INFO 2023-10-01 20:18:39,890 "GET /shop/ HTTP/1.1" 200 1919 basehttp 5499 log_message
WARNING 2023-10-01 20:18:39,965 "GET /static/hwapp1/css/style.css HTTP/1.1" 404 1819 basehttp 5499 log_message
INFO 2023-10-01 20:19:28,510 "GET /shop/ HTTP/1.1" 200 1912 basehttp 5499 log_message
INFO 2023-10-01 20:19:28,570 "GET /static/css/style.css HTTP/1.1" 200 362 basehttp 5499 log_message
INFO 2023-10-01 20:19:31,069 "GET /shop/ HTTP/1.1" 200 1912 basehttp 5499 log_message
INFO 2023-10-01 20:19:42,937 "GET /random/ HTTP/1.1" 200 1496 basehttp 5499 log_message
INFO 2023-10-01 20:19:44,585 "GET /blog/ HTTP/1.1" 200 1436 basehttp 5499 log_message
INFO 2023-10-01 20:21:06,301 "GET /blog/ HTTP/1.1" 200 1436 basehttp 5499 log_message
INFO 2023-10-01 20:21:07,240 "GET /blog/ HTTP/1.1" 200 1436 basehttp 5499 log_message
INFO 2023-10-01 20:21:07,982 "GET /blog/ HTTP/1.1" 200 1436 basehttp 5499 log_message
INFO 2023-10-01 20:21:50,294 Index page accessed views 5499 index
INFO 2023-10-01 20:21:50,295 "GET /me/ HTTP/1.1" 200 1114 basehttp 5499 log_message
INFO 2023-10-01 20:21:50,427 "GET /static/me/images/roger_home.png HTTP/1.1" 304 0 basehttp 5499 log_message
INFO 2023-10-01 20:21:54,759 "GET /blog/ HTTP/1.1" 200 1436 basehttp 5499 log_message
INFO 2023-10-01 20:22:22,797 "GET /random/ HTTP/1.1" 200 1496 basehttp 5499 log_message
INFO 2023-10-01 20:22:52,956 Watching for file changes with StatReloader autoreload 5713 run_with_reloader
INFO 2023-10-01 20:22:55,166 "GET /random/ HTTP/1.1" 200 1496 basehttp 5713 log_message
INFO 2023-10-01 20:22:56,229 "GET /random/ HTTP/1.1" 200 1496 basehttp 5713 log_message
INFO 2023-10-01 20:23:39,975 "GET /blog/ HTTP/1.1" 200 1436 basehttp 5713 log_message
INFO 2023-10-01 20:23:41,961 "GET /blog/articles/ HTTP/1.1" 200 4030 basehttp 5713 log_message
INFO 2023-10-01 20:25:26,654 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5713 trigger_reload
INFO 2023-10-01 20:25:27,152 Watching for file changes with StatReloader autoreload 5742 run_with_reloader
INFO 2023-10-01 20:27:06,250 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5742 trigger_reload
INFO 2023-10-01 20:27:06,676 Watching for file changes with StatReloader autoreload 5775 run_with_reloader
INFO 2023-10-01 20:27:07,477 "GET /blog/ HTTP/1.1" 200 1436 basehttp 5775 log_message
INFO 2023-10-01 20:27:08,568 "GET /blog/articles/ HTTP/1.1" 200 8170 basehttp 5775 log_message
INFO 2023-10-01 20:27:16,962 "GET /blog/articles/add_new_article HTTP/1.1" 200 8170 basehttp 5775 log_message
INFO 2023-10-01 20:29:18,349 /home/olgatorres/PycharmProjects/Django/testproject/blog/urls.py changed, reloading. autoreload 5775 trigger_reload
INFO 2023-10-01 20:29:18,927 Watching for file changes with StatReloader autoreload 5803 run_with_reloader
INFO 2023-10-01 20:29:19,361 "GET /blog/ HTTP/1.1" 200 1603 basehttp 5803 log_message
ERROR 2023-10-01 20:29:20,805 Internal Server Error: /blog/articles/author/Name1 log 5803 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 62, in article_by_author
    author = get_object_or_404(Author, name=name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/db/models/query.py", line 640, in get
    raise self.model.MultipleObjectsReturned(
blog.models.Author.MultipleObjectsReturned: get() returned more than one Author -- it returned 2!
ERROR 2023-10-01 20:29:20,807 "GET /blog/articles/author/Name1 HTTP/1.1" 500 76370 basehttp 5803 log_message
ERROR 2023-10-01 20:29:40,885 Internal Server Error: /blog/articles/author/Name1 log 5803 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 62, in article_by_author
    author = get_object_or_404(Author, name=name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/db/models/query.py", line 640, in get
    raise self.model.MultipleObjectsReturned(
blog.models.Author.MultipleObjectsReturned: get() returned more than one Author -- it returned 2!
ERROR 2023-10-01 20:29:40,887 "GET /blog/articles/author/Name1 HTTP/1.1" 500 76370 basehttp 5803 log_message
ERROR 2023-10-01 20:29:41,864 Internal Server Error: /blog/articles/author/Name1 log 5803 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 62, in article_by_author
    author = get_object_or_404(Author, name=name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/db/models/query.py", line 640, in get
    raise self.model.MultipleObjectsReturned(
blog.models.Author.MultipleObjectsReturned: get() returned more than one Author -- it returned 2!
ERROR 2023-10-01 20:29:41,865 "GET /blog/articles/author/Name1 HTTP/1.1" 500 76370 basehttp 5803 log_message
ERROR 2023-10-01 20:29:44,504 Internal Server Error: /blog/articles/author/Name1 log 5803 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 62, in article_by_author
    author = get_object_or_404(Author, name=name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/db/models/query.py", line 640, in get
    raise self.model.MultipleObjectsReturned(
blog.models.Author.MultipleObjectsReturned: get() returned more than one Author -- it returned 2!
ERROR 2023-10-01 20:29:44,506 "GET /blog/articles/author/Name1 HTTP/1.1" 500 76370 basehttp 5803 log_message
ERROR 2023-10-01 20:29:56,060 Internal Server Error: /blog/articles/author/Name1 log 5803 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 62, in article_by_author
    author = get_object_or_404(Author, name=name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/db/models/query.py", line 640, in get
    raise self.model.MultipleObjectsReturned(
blog.models.Author.MultipleObjectsReturned: get() returned more than one Author -- it returned 2!
ERROR 2023-10-01 20:29:56,061 "GET /blog/articles/author/Name1 HTTP/1.1" 500 76370 basehttp 5803 log_message
ERROR 2023-10-01 20:30:00,995 Internal Server Error: /blog/articles/author/Name1 log 5803 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 62, in article_by_author
    author = get_object_or_404(Author, name=name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/db/models/query.py", line 640, in get
    raise self.model.MultipleObjectsReturned(
blog.models.Author.MultipleObjectsReturned: get() returned more than one Author -- it returned 2!
ERROR 2023-10-01 20:30:00,997 "GET /blog/articles/author/Name1 HTTP/1.1" 500 76370 basehttp 5803 log_message
INFO 2023-10-01 20:30:04,251 "GET /blog/ HTTP/1.1" 200 1603 basehttp 5803 log_message
ERROR 2023-10-01 20:30:06,728 Internal Server Error: /blog/articles/author/Name5 log 5803 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 62, in article_by_author
    author = get_object_or_404(Author, name=name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/db/models/query.py", line 640, in get
    raise self.model.MultipleObjectsReturned(
blog.models.Author.MultipleObjectsReturned: get() returned more than one Author -- it returned 2!
ERROR 2023-10-01 20:30:06,728 "GET /blog/articles/author/Name5 HTTP/1.1" 500 76370 basehttp 5803 log_message
INFO 2023-10-01 20:30:17,991 "GET /blog/ HTTP/1.1" 200 1603 basehttp 5803 log_message
INFO 2023-10-01 20:30:18,918 "GET /blog/articles/author/Name7 HTTP/1.1" 200 1536 basehttp 5803 log_message
INFO 2023-10-01 20:33:10,339 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5803 trigger_reload
INFO 2023-10-01 20:33:10,657 Watching for file changes with StatReloader autoreload 5888 run_with_reloader
INFO 2023-10-01 20:34:37,604 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5888 trigger_reload
INFO 2023-10-01 20:34:38,107 Watching for file changes with StatReloader autoreload 5911 run_with_reloader
INFO 2023-10-01 20:34:41,761 "GET /blog/ HTTP/1.1" 200 1603 basehttp 5911 log_message
INFO 2023-10-01 20:34:44,916 "GET /blog/authors/add_new_author/ HTTP/1.1" 200 1853 basehttp 5911 log_message
INFO 2023-10-01 20:38:30,890 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5911 trigger_reload
INFO 2023-10-01 20:38:31,291 Watching for file changes with StatReloader autoreload 5975 run_with_reloader
INFO 2023-10-01 20:40:32,433 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 5975 trigger_reload
INFO 2023-10-01 20:40:32,892 Watching for file changes with StatReloader autoreload 6034 run_with_reloader
INFO 2023-10-01 20:44:08,707 /home/olgatorres/PycharmProjects/Django/testproject/blog/forms.py changed, reloading. autoreload 6034 trigger_reload
INFO 2023-10-01 20:44:09,156 Watching for file changes with StatReloader autoreload 6103 run_with_reloader
INFO 2023-10-01 20:47:18,481 Watching for file changes with StatReloader autoreload 6193 run_with_reloader
INFO 2023-10-01 20:47:28,608 /home/olgatorres/PycharmProjects/Django/testproject/blog/forms.py changed, reloading. autoreload 6193 trigger_reload
INFO 2023-10-01 20:47:31,073 Watching for file changes with StatReloader autoreload 6206 run_with_reloader
INFO 2023-10-01 20:47:34,136 "GET /blog/ HTTP/1.1" 200 1603 basehttp 6206 log_message
INFO 2023-10-01 20:47:36,214 "GET /blog/authors/add_new_author/ HTTP/1.1" 200 1853 basehttp 6206 log_message
ERROR 2023-10-01 20:47:47,130 Internal Server Error: /blog/authors/add_new_author/ log 6206 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 53, in add_author_form
    {'form': form, 'obj': obj})
UnboundLocalError: local variable 'obj' referenced before assignment
ERROR 2023-10-01 20:47:47,131 "POST /blog/authors/add_new_author/ HTTP/1.1" 500 67628 basehttp 6206 log_message
INFO 2023-10-01 20:47:59,511 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 6206 trigger_reload
INFO 2023-10-01 20:47:59,928 Watching for file changes with StatReloader autoreload 6225 run_with_reloader
INFO 2023-10-01 20:48:01,462 "POST /blog/authors/add_new_author/ HTTP/1.1" 200 1965 basehttp 6225 log_message
INFO 2023-10-01 20:48:20,526 Add new author name='asd', surname='s' views 6225 add_author_form
INFO 2023-10-01 20:48:20,539 "POST /blog/authors/add_new_author/ HTTP/1.1" 200 1904 basehttp 6225 log_message
INFO 2023-10-01 20:50:47,169 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 6225 trigger_reload
INFO 2023-10-01 20:50:47,696 Watching for file changes with StatReloader autoreload 6261 run_with_reloader
INFO 2023-10-01 20:50:48,417 Add new author name='asd', surname='s' views 6261 add_author_form
INFO 2023-10-01 20:50:48,436 "POST /blog/authors/add_new_author/ HTTP/1.1" 200 1936 basehttp 6261 log_message
INFO 2023-10-01 20:51:38,181 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 6261 trigger_reload
INFO 2023-10-01 20:51:38,659 Watching for file changes with StatReloader autoreload 6309 run_with_reloader
INFO 2023-10-01 20:54:58,590 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 6309 trigger_reload
INFO 2023-10-01 20:54:59,184 Watching for file changes with StatReloader autoreload 6350 run_with_reloader
INFO 2023-10-01 20:55:04,402 Add new author name='asd', surname='s' views 6350 add_author_form
INFO 2023-10-01 20:55:04,423 "POST /blog/authors/add_new_author/ HTTP/1.1" 200 1891 basehttp 6350 log_message
INFO 2023-10-01 20:55:07,115 "GET /blog/ HTTP/1.1" 200 1603 basehttp 6350 log_message
INFO 2023-10-01 20:55:08,421 "GET /blog/authors/add_new_author/ HTTP/1.1" 200 1852 basehttp 6350 log_message
INFO 2023-10-01 20:58:09,061 /home/olgatorres/PycharmProjects/Django/testproject/blog/forms.py changed, reloading. autoreload 6350 trigger_reload
INFO 2023-10-01 20:58:09,513 Watching for file changes with StatReloader autoreload 6427 run_with_reloader
INFO 2023-10-01 21:03:26,950 Watching for file changes with StatReloader autoreload 6627 run_with_reloader
INFO 2023-10-01 21:07:19,641 Watching for file changes with StatReloader autoreload 6689 run_with_reloader
INFO 2023-10-01 21:07:23,815 "GET /blog/ HTTP/1.1" 200 1603 basehttp 6689 log_message
WARNING 2023-10-01 21:07:26,288 Not Found: /blog/articles/add_new_article/ log 6689 log_response
WARNING 2023-10-01 21:07:26,289 "GET /blog/articles/add_new_article/ HTTP/1.1" 404 3926 basehttp 6689 log_message
INFO 2023-10-01 21:08:16,439 /home/olgatorres/PycharmProjects/Django/testproject/blog/urls.py changed, reloading. autoreload 6689 trigger_reload
INFO 2023-10-01 21:08:16,871 Watching for file changes with StatReloader autoreload 6712 run_with_reloader
WARNING 2023-10-01 21:08:17,390 Not Found: /blog/articles/add_new_article/ log 6712 log_response
WARNING 2023-10-01 21:08:17,391 "GET /blog/articles/add_new_article/ HTTP/1.1" 404 3926 basehttp 6712 log_message
WARNING 2023-10-01 21:08:18,089 Not Found: /blog/articles/add_new_article/ log 6712 log_response
WARNING 2023-10-01 21:08:18,089 "GET /blog/articles/add_new_article/ HTTP/1.1" 404 3926 basehttp 6712 log_message
WARNING 2023-10-01 21:08:20,340 Not Found: /blog/articles/add_new_article/ log 6712 log_response
WARNING 2023-10-01 21:08:20,341 "GET /blog/articles/add_new_article/ HTTP/1.1" 404 3926 basehttp 6712 log_message
INFO 2023-10-01 21:08:45,296 /home/olgatorres/PycharmProjects/Django/testproject/blog/urls.py changed, reloading. autoreload 6712 trigger_reload
INFO 2023-10-01 21:08:45,721 Watching for file changes with StatReloader autoreload 6723 run_with_reloader
INFO 2023-10-01 21:08:46,156 "GET /blog/articles/add_new_article/ HTTP/1.1" 200 3038 basehttp 6723 log_message
INFO 2023-10-01 21:09:01,494 Add new article author=<Author: Name2 Surname2>, title='ss' views 6723 add_article_form
ERROR 2023-10-01 21:09:01,528 Internal Server Error: /blog/articles/add_new_article/ log 6723 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 204, in _get_response
    self.check_response(response, callback)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 332, in check_response
    raise ValueError(
ValueError: The view blog.views.add_article_form didn't return an HttpResponse object. It returned None instead.
ERROR 2023-10-01 21:09:01,529 "POST /blog/articles/add_new_article/ HTTP/1.1" 500 68759 basehttp 6723 log_message
INFO 2023-10-01 21:09:17,177 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 6723 trigger_reload
INFO 2023-10-01 21:09:17,584 Watching for file changes with StatReloader autoreload 6750 run_with_reloader
INFO 2023-10-01 21:09:21,594 Add new article author=<Author: Name2 Surname2>, title='ss' views 6750 add_article_form
INFO 2023-10-01 21:09:21,618 "POST /blog/articles/add_new_article/ HTTP/1.1" 200 3076 basehttp 6750 log_message
INFO 2023-10-01 21:17:13,522 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 6750 trigger_reload
INFO 2023-10-01 21:17:14,015 Watching for file changes with StatReloader autoreload 7048 run_with_reloader
INFO 2023-10-01 21:17:14,991 "GET /blog/ HTTP/1.1" 200 1603 basehttp 7048 log_message
INFO 2023-10-01 21:17:17,370 "GET /blog/articles/ HTTP/1.1" 200 8318 basehttp 7048 log_message
INFO 2023-10-01 21:17:19,262 "GET /blog/article/5/ HTTP/1.1" 200 944 basehttp 7048 log_message
ERROR 2023-10-01 21:20:43,524 Internal Server Error: /blog/article/5/ log 7048 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endblock'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 101, in article_content
    return render(request,
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/template/base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 16: 'endblock', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
ERROR 2023-10-01 21:20:43,527 "GET /blog/article/5/ HTTP/1.1" 500 181631 basehttp 7048 log_message
INFO 2023-10-01 21:20:56,953 "GET /blog/article/5/ HTTP/1.1" 200 1328 basehttp 7048 log_message
INFO 2023-10-01 21:25:12,062 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 7048 trigger_reload
INFO 2023-10-01 21:25:12,494 Watching for file changes with StatReloader autoreload 7172 run_with_reloader
INFO 2023-10-01 21:25:13,112 "GET /blog/article/5/ HTTP/1.1" 200 1608 basehttp 7172 log_message
INFO 2023-10-01 21:26:13,589 "GET /blog/article/5/ HTTP/1.1" 200 1664 basehttp 7172 log_message
INFO 2023-10-01 21:26:46,333 "GET /blog/article/5/ HTTP/1.1" 200 1692 basehttp 7172 log_message
INFO 2023-10-01 21:27:03,648 "GET /blog/article/5/ HTTP/1.1" 200 1622 basehttp 7172 log_message
INFO 2023-10-01 21:27:04,472 "GET /blog/article/5/ HTTP/1.1" 200 1622 basehttp 7172 log_message
INFO 2023-10-01 21:30:12,243 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 7172 trigger_reload
INFO 2023-10-01 21:30:12,634 Watching for file changes with StatReloader autoreload 7230 run_with_reloader
ERROR 2023-10-01 21:31:44,723 Internal Server Error: /blog/article/5/ log 7230 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 100, in article_content
    comments = Comment.objects.filter(article=article).sort_by
AttributeError: 'QuerySet' object has no attribute 'sort_by'
ERROR 2023-10-01 21:31:44,724 "GET /blog/article/5/ HTTP/1.1" 500 66242 basehttp 7230 log_message
ERROR 2023-10-01 21:31:45,732 Internal Server Error: /blog/article/5/ log 7230 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 100, in article_content
    comments = Comment.objects.filter(article=article).sort_by
AttributeError: 'QuerySet' object has no attribute 'sort_by'
ERROR 2023-10-01 21:31:45,733 "GET /blog/article/5/ HTTP/1.1" 500 66242 basehttp 7230 log_message
INFO 2023-10-01 21:32:25,321 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 7230 trigger_reload
INFO 2023-10-01 21:32:25,739 Watching for file changes with StatReloader autoreload 7273 run_with_reloader
ERROR 2023-10-01 21:32:26,303 Internal Server Error: /blog/article/5/ log 7273 log_response
Traceback (most recent call last):
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/olgatorres/PycharmProjects/Django/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/olgatorres/PycharmProjects/Django/testproject/blog/views.py", line 100, in article_content
    comments = Comment.objects.filter(article=article).sortby(added_date)
AttributeError: 'QuerySet' object has no attribute 'sortby'
ERROR 2023-10-01 21:32:26,304 "GET /blog/article/5/ HTTP/1.1" 500 66260 basehttp 7273 log_message
INFO 2023-10-01 21:33:24,432 /home/olgatorres/PycharmProjects/Django/testproject/blog/views.py changed, reloading. autoreload 7273 trigger_reload
INFO 2023-10-01 21:33:24,918 Watching for file changes with StatReloader autoreload 7553 run_with_reloader
INFO 2023-10-01 21:33:26,789 "GET /blog/article/5/ HTTP/1.1" 200 1903 basehttp 7553 log_message
INFO 2023-10-01 21:36:03,958 /home/olgatorres/PycharmProjects/Django/testproject/blog/forms.py changed, reloading. autoreload 7553 trigger_reload
INFO 2023-10-01 21:36:04,541 Watching for file changes with StatReloader autoreload 7596 run_with_reloader
